templates/vue/src/components/NodeModal.vue

```diff
@@ -97,6 +98,14 @@
             </slick-list>
           </div>
         </b-tab>
+        <b-tab
+          v-if="showMap"
+          title="Coordinate"
```

Comment:
Please change the tab to be called "Map". Please also change the variable names to be plural i.e. `coordinates` and the component to `coordinates-form`.